react/features/base/react/components/native/Container.js

```diff
@@ -27,6 +27,7 @@ export default class Container<P: Props> extends AbstractContainer<P> {
             accessibilityLabel,
             accessible,
             onClick,
+            onLongPress,
```

Comment:
The logic below doesn't account for the case in which there is only a long press handler but not an onlick handler. touchFeedback should have a default value of `Boolean(onClick || onLongPress)`